ibm/resource_ibm_pi_network_port_attach.go

```diff
@@ -40,18 +40,21 @@ func resourceIBMPINetworkPortAttach() *schema.Resource {
 			},
 
 			helpers.PIInstanceName: {
-				Type:     schema.TypeString,
-				Required: true,
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Instance name to attach the network port to",
 			},
 
 			helpers.PINetworkName: {
-				Type:     schema.TypeString,
-				Required: true,
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Network Name - This is the subnet name  in the Cloud instance",
 			},
 
 			"description": {
-				Type:     schema.TypeString,
```

Comment:
Description is optional field...can we add the helper string pi_description and in create don't set the default description if it's not set by user
```
if d.Get(description) == "" {
		description = "Testing from Terraform"
	}
```